libcrux-ml-kem/src/serialize.rs

```diff
@@ -1,20 +1,23 @@
 use crate::{
-    constants::{BYTES_PER_RING_ELEMENT, SHARED_SECRET_SIZE},
-    hax_utils::hax_debug_assert,
+    constants::{COEFFICIENTS_IN_RING_ELEMENT, BYTES_PER_RING_ELEMENT, SHARED_SECRET_SIZE},
     helper::cloop,
     polynomial::{PolynomialRingElement, VECTORS_IN_RING_ELEMENT},
     vector::{decompress_1, to_unsigned_representative, Operations},
 };
 
+use hax_lib::*;
```

Comment:
This needs a `cfg` guard.

Reply:
What is supposed to be inside `cfg` expression?